joker/sacrificial_lamb.lua

```diff
@@ -0,0 +1,45 @@
+SMODS.Joker {
+    key = 'sacrificial_lamb',
+    loc_txt = {
+        name = "Sacrificial Lamb",
+        text = {
+            "Gains {C:mult}+5 Mult{} when",
+            "for each card {C:attention}sold{}",
+            "{C:inactive}(Currently gives {}{C:mult}+#1#{} {C:inactive}Mult){}"
```

Comment:
This effect should be on every card "destroyed", not sold

Reply:
- "when for each card sold" grammar aint grammar'ing
- As a tip, take a look at other jokers already in the game to get a more natural wording of the functionality of the joker.
Namely, adapting from campfire:

This Joker gains +5 Mult for each card or Joker destroyed

This also includes the highlighting of the important bits, of course.